pertpy/tools/_perturbation_space/_simple.py

```diff
@@ -168,6 +174,48 @@ def compute(
 
         return ps_adata
 
+    @_doc_params(common_plot_args=doc_common_plot_args)
+    def plot_psbulk_samples(
+        self,
+        adata: AnnData,
+        groupby: str,
+        show: bool = True,
+        save: str | bool = False,
+        return_fig: bool = False,
+        **kwargs,
+    ) -> Figure | None:
+        """Plot the pseudobulk samples of an AnnData object. It uses Decoupler implementation.
```

Comment:
It's a convention that there should only ever be one sentence in the first line of a docstring aka the summary sentence. I think it's fine to remove the sentence.
```suggestion
        """Plot the pseudobulk samples of an AnnData object.
```